src/picker/styles/variables.scss

```
@@ -14,6 +14,7 @@
   --num-columns: 8; /* How many columns to display in the emoji grid */
   --outline-size: 2px; /* Focus outline width */
   --border-size: 1px; /* Width of border used in most of the picker */
+  --border-radius: 0rem; /* Border radius of the entire picker */
```

```suggestion
  --border-radius: 0; /* Border radius of the entire picker */
```

No need for the `rem`.